frappe/public/js/frappe/misc/datetime.js

```diff
@@ -156,5 +156,5 @@ $.extend(frappe.datetime, {
 });
 
 // globals (deprecate)
-var date = dateutil = frappe.datetime;
-var get_today = frappe.datetime.get_today;
+// var date = dateutil = frappe.datetime;
+// var get_today = frappe.datetime.get_today;
```

Comment:
@netchampfaris, Can we keep this for backward capability, these methods might be used in custom script or custom app